FIX: revisit preprocessed data QC (#440)
TheAxonLab/hcph-sops: docs/processing/preprocessing.md

## Executing *fMRIPrep*
Because *fMRIPrep* creates a single anatomical reference for all sessions, we generate such reference first by setting the `--anat-only` flag.
If that *fMRIPrep* execution finishes successfully, the anatomical processing outcomes will be stored in the output folder.

> We will then run one *fMRIPrep* process for each dataset's session, which is the recommended way for datasets with a large number of sessions (e.g., more than 
> six sessions).

Reviewer: Why this line break?

```suggestion
We will then run one *fMRIPrep* process for each dataset's session, which is the recommended way for datasets with a large number of sessions (e.g., more than  six sessions).
```

Author: I'm not sure why I have many seemingly random line break ?!

{% endfilter %}
        ```

> - [ ] Once the anatomical workflow ran successfully, submit a *job array* with one scanning session each with the `--bids-filter-file` argument selecting the
>     corresponding session, and point the `--fs-subjects-dir` argument to the folder where *FreeSurfer* results were stored.

Reviewer: Our markdown is configured to break lines when you add them, so this will render weirdly (I'm 90% sure of this):
```suggestion
- [ ] Once the anatomical workflow ran successfully, submit a *job array* with one scanning session each with the `--bids-filter-file` argument selecting the corresponding session, and point the `--fs-subjects-dir` argument to the folder where *FreeSurfer* results were stored.
```

    
    - [ ] Check the "Errors" section of the visual report.
    - [ ] Check the `log/` folder corresponding to the *fMRIPrep* run, carefully ensuring no errors were missed out on the reports.

>     - [ ] Search for associated keywords in the [issues on *fMRIPrep*'s GitHub repository](https://github.com/nipreps/fmriprep/issues) if the solution remains 
>         unclear after the first assessment; it is likely someone else experienced the same problem before you and reported it.

Reviewer: ```suggestion
    - [ ] Search for associated keywords in the [issues on *fMRIPrep*'s GitHub repository](https://github.com/nipreps/fmriprep/issues) if the solution remains unclear after the first assessment; it is likely someone else experienced the same problem before you and reported it.
```

## Visualizing *fMRIPrep*'s individual reports

Following our protocols<sup>[1]</sup>, the quality of unprocessed images MUST be assessed [before](../data-management/mriqc.md#visualizing-mriqcs-individual-reports) and after preprocessing to verify it did not go awry.

### Anatomical preprocessing assessment

- [ ] Open the *fMRIPrep* anatomical report on a current Web Browser (*Google Chrome* is preferred).
- [ ] Assess the "Summary" section and apply the [QA/QC criteria](qaqc-criteria-preprocessed.md#summary).
- [ ] Assess the "Anatomical conformation" section and apply the [QA/QC criteria](qaqc-criteria-preprocessed.md#anatomical-conformation).
- [ ] Assess the mosaic showing the calculated brain mask and brain tissue segmentation, and apply the [QA/QC criteria](qaqc-criteria-preprocessed.md#brain-mask-and-brain-tissue-segmentation-of-the-t1w).
- [ ] Visualize the spatial normalization flickering mosaic, and apply the [QA/QC criteria](qaqc-criteria-preprocessed.md#spatial-normalization-of-the-anatomical-t1w-reference).

>     Flickering between the subject and the template space is active while hovering your mouse on the mosaic area.

Reviewer: ```suggestion
    Flickering between the subject and the template space is active while your mouse pointer hovers the mosaic area.
```

- [ ] Assess the surface reconstruction mosaic, and apply the [QA/QC criteria](qaqc-criteria-preprocessed.md#surface-reconstruction).

> - [ ] Visualize the first section entitled `Summary` and apply the [QA/QC criteria](qaqc-criteria-preprocessed.md#summary).

Reviewer: ```suggestion
- [ ] Visualize the first section entitled *Summary* and apply the [QA/QC criteria](qaqc-criteria-preprocessed.md#summary).
```

    !!! warning "Running through the visual report does not have to be executed in a fixed order"

>         While we describe the run through the report as linear, often inspecting other reportlets can help make a decision about the exclusion criteria related 
>         to the reportlet at hand. 

Reviewer: linebreak
```suggestion
        While we describe the run through the report as linear, often inspecting other reportlets can help make a decision about the exclusion criteria related to the reportlet at hand. 
```

Please update to follow the directions we have sketched in the previous PR to qc of unprocessed data (there's a similar admonition that I have edited for clarity).

    Except if the image is [extremely distorted](qaqc-criteria.md#task-fmri-exclusion-criteria), we are not excluding QCT scans because we will leverage those  
    images to evaluate the quality of fMRI scans and derived constructs throughout the whole analysis pipeline. We are however going through the *MRIQC* reports 
    to train our eye, anticipate issues that might be affecting RSfMRI and flag the corresponding sessions.

- [ ] Go through the report section of the QCT
    - [ ] Scrutinize the textual summary and apply the [QA/QC criteria](qaqc-criteria-preprocessed.md#textual-summary).
    - [ ] Visualize the T2* map mosaic, and apply the [QA/QC criteria](qaqc-criteria-preprocessed.md#t2-map).
    - [ ] Check the T2* gray-matter intensity histogram, and apply the [QA/QC criteria](qaqc-criteria-preprocessed.md#t2-gray-matter-values).
    - [ ] Visualize the co-registration flickering mosaic, apply the [QA/QC criteria](qaqc-criteria-preprocessed.md#alignment-of-functional-and-anatomical-mri-data).
        Flickering between T1w and BOLD images is active while hovering your mouse on the mosaic area.

>     - [ ] Skip the next reportlets and go to the BHT section

Reviewer: If this is the same below, please merge and indicate that you sequentially do this on the QCT and then BHT.

- [ ] Scrutinize the textual summary and apply the [QA/QC criteria](qaqc-criteria-preprocessed.md#textual-summary).
- [ ] Visualize the T2* map mosaic, and apply the [QA/QC criteria](qaqc-criteria-preprocessed.md#t2-map).
- [ ] Check the T2* gray-matter intensity histogram, and apply the [QA/QC criteria](qaqc-criteria-preprocessed.md#t2-gray-matter-values).
- [ ] Visualize the co-registration flickering mosaic, apply the [QA/QC criteria](qaqc-criteria-preprocessed.md#alignment-of-functional-and-anatomical-mri-data).
    Flickering between T1w and BOLD images is active while hovering your mouse on the mosaic area.
- [ ] Visualize the next mosaic displaying regions of interest (ROIs) used to estimate the nuisance regressors, and apply the [QA/QC criteria](qaqc-criteria-preprocessed.md#brain-mask-and-anatomicaltemporal-compcor-rois).
- [ ] Visualize the carpet plot and nuisance signals panel, and apply the [QA/QC criteria](qaqc-criteria-preprocessed.md#bold-summary).
- [ ] Visualize the confound correlation heatmap and use it to [choose the regressors](qaqc-criteria-preprocessed.md#correlations-between-nuisance-regressors) 
    you will include in the nuisance regression model.

> 

Reviewer: ```suggestion
```

> - [ ] Finally, verify that no errors are reported within the "Errors" section. If there are, proceed as indicated in ["Not all *fMRIPrep* derivatives were generated"](#executing-fmriprep).

Reviewer: Please edit like suggested in the qc of unprocessed.
```suggestion
- [ ] Finally, verify that no errors are reported within the "Errors" section.
    If there are, proceed as indicated in ["Not all *fMRIPrep* derivatives were generated"](#executing-fmriprep).
```
